.infrastructure/07_ecs.tf

```diff
@@ -25,7 +25,8 @@ data "template_file" "cms_app" {
     access_key_id        = aws_ssm_parameter.cms_access_key_id.arn
     access_key_secret    = aws_ssm_parameter.cms_access_key_secret.arn
     bucket_full_url      = aws_s3_bucket.cms_medialibrary_bucket.bucket_regional_domain_name
-    cdn_url              = "https://${aws_cloudfront_distribution.cms_medialibrary.domain_name}/"
+    cdn_url              = "https://${aws_cloudfront_distribution.cms_medialibrary.domain_name}"
+    aws_bucket_endpoint  = "https://s3.${var.aws_region}.amazonaws.com"
```

Comment:
Why are we using path-style URLs instead of virtual-hosted–style? The [path-style will be deprecated](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#:~:text=For%20more%20information%2C%20see%20Amazon%20S3%20Path%20Deprecation%20Plan%20%E2%80%93%20The,in%20the%20AWS%20News%20Blog.).

Reply:
@simosalsi: In Strapi, uou can use the format "https://${bucket_name}.s3.${var.aws_region}.amazonaws.com" ?

Reply:
Opened PR to verify use of this style in Strapi